cli/config/config.go

```diff
@@ -1,28 +1,30 @@
 package config
 
 import (
-	"io/ioutil"
 	"os"
 	"os/user"
 	"strings"
 	"time"
 
-	"github.com/goccy/go-yaml"
-	"github.com/sirupsen/logrus"
+	"github.com/defenseunicorns/zarf/cli/internal/utils"
 )
 
 const K3sBinary = "/usr/local/bin/k3s"
-const K3sChartPath = "/var/lib/rancher/k3s/server/static/charts"
-const K3sManifestPath = "/var/lib/rancher/k3s/server/manifests"
-const K3sImagePath = "/var/lib/rancher/k3s/agent/images"
+
 const PackageInitName = "zarf-init.tar.zst"
 const PackagePrefix = "zarf-package-"
-const ZarfLocalIP = "127.0.0.1"
 const ZarfGitUser = "zarf-git-user"
+const ZarfStatePath = ".zarf-state.yaml"
 
 var CLIVersion = "unset"
+var config ZarfPackage
```

Comment:
Apparently I can't leave comments on lines the PR doesn't think changed.  After rebasing onto master ZarfBuildData() around line 54 needs to return `config.Build` instead of `config.Package`.